packages/boot/tools/ibc/ics27-1.md

```diff
@@ -0,0 +1,113 @@
+
+# ICS-27 Interchain Accounts
+
+Sequence diagrams for Interchain Accounts based on [ics-027-interchain-accounts/README.md#9ffb1d2](https://github.com/cosmos/ibc/blob/9ffb1d26d3018b6efda546189ec7e43d56d23da3/spec/app/ics-027-interchain-accounts/README.md).
```

Comment:
👏 

consider making the file `README.md` so it will appear when someone navigates to the directory path